cns/requestcontroller/requestcontrollerintreface.go

```diff
@@ -3,17 +3,12 @@ package requestcontroller
 import (
 	"context"
 
-	"github.com/Azure/azure-container-networking/cns"
 	nnc "github.com/Azure/azure-container-networking/nodenetworkconfig/api/v1alpha"
 )
 
 // RequestController interface for cns to interact with the request controller
```

Comment:
Why would CNS will call into RequestController to InitCNS?

Reply:
I was thinking CNS would create a new request controller obj, call InitCNS on that obj, and then call StartRequestController on that obj.
How are you envisioning it?